js/src/offcanvas.js

```diff
@@ -198,12 +211,13 @@ class Offcanvas extends BaseComponent {
         return
       }
 
-      if (!this._config.keyboard) {
-        EventHandler.trigger(this._element, EVENT_HIDE_PREVENTED)
+      if (this._config.keyboard) {
```

Comment:
I suppose this change is out of scope. Right?

Reply:
Yes, out of scope and only a styling change. I think it's better since it makes this piece of code identical to the `modal` keydown handler.

Reply:
Let's address it on another PR

Reply:
> Let's address it on another PR

Separated this refactor into #38025

Reply:
Se revert it form here